.github/ISSUE_TEMPLATE/git-failure-due-to-authentication.md

```diff
@@ -0,0 +1,38 @@
+---
+name: Git failure due to authentication
+about: Create a report to help us improve
+title: ''
+labels: ''
+assignees: ''
+
+---
+
+**Which Version of GCM are you using ?**
+You can see the GCM version by setting the env variable GCM_TRACE=1 and performing a command.
```

Comment:
nit: I think this must be a Git network operation, not any command?

Reply:
I thought about this more. Let's give more specific instructions, like:
`From a command prompt, run `git credential-manager version` and paste the output.`